tests/unit/test_parsers.py

```diff
@@ -100,23 +100,60 @@ def test_complete_parsing(parser_class, raw_file, results_file):
 
 
 @pytest.mark.parametrize(
-    "parser_class, raw_file, exception",
+    "parser_class, raw_file, exception, error_message",
     [
         # ICal
-        (ICal, Path(dir_path, "data", "ical", "ical_no_account"), MissingMandatoryFields,),
-        (ICal, Path(dir_path, "data", "ical", "ical_no_maintenance_id"), MissingMandatoryFields,),
-        (ICal, Path(dir_path, "data", "ical", "ical_no_stamp"), ParsingError,),
-        (ICal, Path(dir_path, "data", "ical", "ical_no_start"), ParsingError,),
-        (ICal, Path(dir_path, "data", "ical", "ical_no_end"), ParsingError,),
+        (
+            ICal,
+            Path(dir_path, "data", "ical", "ical_no_account"),
+            MissingMandatoryFields,
+            "1 validation error for Maintenance\naccount\n  String is empty or 'None' (type=value_error)",
+        ),
+        (
+            ICal,
+            Path(dir_path, "data", "ical", "ical_no_maintenance_id"),
+            MissingMandatoryFields,
+            "1 validation error for Maintenance\nmaintenance_id\n  String is empty or 'None' (type=value_error)",
```

Comment:
For readability, consider using triple-quoted strings with literal newlines, e.g.:

```suggestion
            """\
1 validation error for Maintenance
maintenance_id
  String is empty or 'None' (type=value_error)""",
```